storage/gcloud/aio/storage/storage.py

```diff
@@ -14,6 +15,9 @@
 STORAGE_UPLOAD_API_ROOT = 'https://www.googleapis.com/upload/storage/v1/b'
 READ_WRITE_SCOPE = 'https://www.googleapis.com/auth/devstorage.read_write'
 
+MAX_CONTENT_LENGTH_SIMPLE_UPLOAD = 5242880  # 5 MB
```

Comment:
Rather than commenting that this number is equal to 5 MB, it would probably be clearer to set:
```
MAX_CONTENT_LENGTH_SIMPLE_UPLOAD = 5 * 1024 * 1024
```